src/pages/library/DataBrowserDetails.js

```diff
@@ -31,22 +36,130 @@ const getSnapshot = id => new Promise(resolve => setTimeout(() => {
     subjects: 10,
     dataType: 'RNA Seq',
     keepCollapsed: true,
-    locked: false,
-    files: 15,
-    fileSize: 4432
+    locked: true,
+    donor: {
+      size: 1202,
+      modality: 'Exome'
+    },
+    sample: {
+      size: 10593
+    },
+    files: {
+      count: 15,
+      size: 4432,
+      types: {
+        bam: 42,
+        cram: 11,
+        vcf: 20,
+        tar: 1
+      }
+    },
+    releasePolicy: 'GRU',
+    region: 'Multi-region - US',
+    cloudProvider: 'Azure',
+    contributors: ['Erin Dogra', 'Jinzhou Zuanoh', 'Donna Bechard', 'Yim Lang Ching', 'David Smith', 'Peter Hanna', 'Pupsa Shape', 'Joel Szabo']
   })
-}, 2000))
+}))
 
 const MainContent = snapshot => {
   return div({ style: { ...styles.content, width: '100%', marginTop: 0 } }, [
-    h1({ style: { marginTop: 0 } }, [snapshot.name]),
-    div({ style: { whiteSpace: 'pre-wrap' } }, [JSON.stringify(snapshot, null, 2)])
+    h1([snapshot.name]),
+    div([
+      div([snapshot.description]),
+      div({ style: { marginTop: 30 } }, [JSON.stringify(snapshot)]),
+      h2({ className: 'sr-only' }, ['Snapshot Sources']),
+      div({ style: { display: 'flex', width: '100%' } }, [
+        div({ style: styles.attributesColumn }, [
+          h3({ style: styles.headers }, ['Data release policy']),
+          div([snapshot.releasePolicy])
+        ]), div({ style: styles.attributesColumn }, [
+          h3({ style: styles.headers }, ['Region']),
+          div([snapshot.region])
+        ]), div({ style: styles.attributesColumn }, [
+          h3({ style: styles.headers }, ['Cloud provider']),
+          div([snapshot.cloudProvider])
+        ])
+      ]), div({ style: { display: 'flex', width: '100%' } }, [
+        div({ style: styles.attributesColumn }, [
+          h3({ style: styles.headers }, ['Contact']),
+          div(['Eric Miron']),
+          div(['University of Chicago Medical Center']),
+          h(Link, { href: `mailto:fakeemail@fake.org` }, ['fakeemail@fake.org'])
+        ]), div({ style: styles.attributesColumn }, [
+          h3({ style: styles.headers }, ['Data curator']),
+          div(['Will add later, after data structure is added'])
+        ]), div({ style: styles.attributesColumn }, [
+          h3({ style: styles.headers }, ['Contributors']),
+          ..._.map(contributor => div(contributor), snapshot.contributors)
+        ]), div({ style: styles.attributesColumn }, [
+          h3({ style: styles.headers }, ['Publications']),
+          div(['Will add later, after data structure is added'])
+        ])
+      ])
```

Comment:
Stylistically, we tend to put the element on a newline after a comma. It makes it a bit easier to see the structure / children. I also tweaked a few lines where some array brackets were missing on the children
```suggestion
      div({ style: { display: 'flex', width: '100%' } }, [
        div({ style: styles.attributesColumn }, [
          h3({ style: styles.headers }, ['Contact']),
          div(['Eric Miron']),
          div(['University of Chicago Medical Center']),
          h(Link, { href: `mailto:fakeemail@fake.org` }, ['fakeemail@fake.org'])
        ]),
        div({ style: styles.attributesColumn }, [
          h3({ style: styles.headers }, ['Data curator']),
          div(['Will add later, after data structure is added'])
        ]),
        div({ style: styles.attributesColumn }, [
          h3({ style: styles.headers }, ['Contributors']),
          _.map(contributor => div([contributor]), snapshot.contributors)
        ]),
        div({ style: styles.attributesColumn }, [
          h3({ style: styles.headers }, ['Publications']),
          div(['Will add later, after data structure is added'])
        ])
      ])
```